cluster/local/README.md

```diff
@@ -70,6 +70,28 @@ For complete list of subcommands supported by `microk8s.sh`, run:
 cluster/local/microk8s.sh
 ```
 
+## Run locally out-of-cluster
+
+For convenience and speed of development, it can be a good option to run
+crossplane locally, out-of-cluster. To do that, there is a target in the
+Makefile:
+
+```
+make run
```

Comment:
Someone wanting to run the Stack Manager will need to supply arguments to `make run` which must be passed along to the binary invocation.

`make ARGS="extension manage" run`

Reply:
That's true. I was intentionally trying to avoid supporting passthrough arguments at first; for now I figured someone could just run it directly by hand with different arguments if they wanted to do that